content/en/monitors/create/types/ci.md

```diff
@@ -63,6 +70,63 @@ The following example is of a pipeline error rate monitor using a formula that c
 
 <div class="alert alert-info"><strong>Note</strong>: Only up to 2 queries can be used to build the evaluation formula per monitor.</div>
 
+{{% /tab %}}
+{{% tab "Tests" %}}
+
+### Define the search query
+
+1. Construct a search query using the same logic as a CI Test explorer search. For example, we can search failed tests for `main` branch using the following query: `@test.status:fail @git.branch:main`
+2. Choose to monitor over a CI Test event count, facet, or measure:
+    * **Monitor over a CI Test event count**: Use the search bar (optional) and do **not** select a facet or measure. Datadog evaluates the number of CI Test events over a selected time frame, then compares it to the threshold conditions.
+    * **Monitor over a dimension**: If a dimension (qualitative facet) is selected, the monitor alerts over the `Unique value count` of the facet.
+    * **Monitor over a measure**: If a measure (quantitative facet) is selected, the monitor alerts over the numerical value of the CI Test facet (similar to a metric monitor) and aggregation needs to be selected (`min`, `avg`, `sum`, `median`, `pc75`, `pc90`, `pc95`, `pc98`, `pc99`, or `max`).
+3. Group CI Test events by multiple dimensions (optional):
+   All CI Test events matching the query are aggregated into groups based on the value of up to four facets.
+4. Configure the alerting grouping strategy (optional):
+   * If the query has a `group by`, you will receive an alert for every source according to the group parameters. An alerting event is generated for each group that meets the set conditions. For example, you could group a query by `@test.full_name` (which is a combination of a test bundle, test suite and test name, e.g. `MyBundle.MySuite.myTest`) to receive a separate alert for each CI Test full name when the number of errors is high.
+
+{{< img src="monitors/monitor_types/ci_tests/define-the-search-query.png" alt="A query for CI Status:Error that is being set to group by Pipeline Name" style="width:80%;" >}}
+
+#### Test runs with different parameters or configurations
+If you run tests with the same test full name, but different test parameters or configurations, it's recommended to use `@test.fingerprint` in the monitor group by. This way alerts will trigger for test runs with specific test parameters or configurations.
+
+For example, if a test with the same full name failed on Chrome browser, but it didn't fail on Firefox, the alert will be triggered for the test run on Chrome.
+Using `@test.full_name` in this case might lead to incorrect behaviour.
+
+Test fingerpint is a hash value calculated using the following attributes:
```

Comment:
Not sure if it's the best place to detail the fingerprint internals. This is not related to monitors exclusively. Also, it can change over time. If we really want to communicate the `test.fingerprint` feature, I'd suggest using a dedicated page in the CI section, and then linking it to that.